### DISCUSSION TITLE
do we still need baresip Google Group?

### DISCUSSION THREAD

**Discussion by juha-h** [baresip/baresip#1874] [General]
Now that there is Discussions opportunity in GiHub, do we still need  baresip Google Group?

I would prefer only one discussion forum and this would be my choice.

**Comment by cspiel1** [comment]
Maybe discussions should be used by contributors, maintainers for future improvements and users for feature requests.
The google group for general questions. A single forum would be also okay for me.

**Comment by alfredh** [comment]
we can use Discussions for baresip discussions.

we need to keep the Google Group as a backup communication channel, in case Github goes down.

**Comment by juha-h** [comment]
Alfred E. Heggestad writes:

> we can use Discussions for baresip discussions.
> 
> we need to keep the Google Group as a backup communication channel, in
> case Github goes down.

OK, fine.
